src/selection.ts

```diff
@@ -1257,7 +1272,7 @@ export class TileSorter
         const rawSortInfo = tile.sorts[this.sortKey];
         // Skip tiles with empty selections
         if (rawSortInfo.indices.length <= 0) {
-          continue
+          continue;
         }
         const sortInfo: SortInfoWithPointer = {
           ...rawSortInfo,
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +1339]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Oops i wanted this behavior but forgot to remove the `return undefined`. Do you want to also have this not throw now? Some of the logic in `initFromK` will have to change in TileSorter to handle `undefined` gracefully. 

I would expect an array method to fail here due to out of bounds but not sure what the javascript best practice is here.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/nomic-ai/deepscatter/168?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Hmm... I think this *won't* throw in userland because select is wrapped inside `DataSelection.get()`, right